src/LearnContent/LearnContent.tsx

```
@@ -0,0 +1,44 @@
+import Head from "next/head";
+import { MDXRemote } from "next-mdx-remote";
+import { Typography } from "hazel-ui/Typography";
+
+import type { LearnPost } from "../lib/learn.js";
+import { Layout } from "../wrappers/Layout.js";
+import { Date } from "../ui/Atoms/Date.js";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)